GEOparse/GEOparse.py

```diff
@@ -388,14 +390,16 @@ def parse_GSM(filepath, entry_name=None):
     return gsm
 
 
-def parse_GPL(filepath, entry_name=None):
+def parse_GPL(filepath, entry_name=None, partial=None):
     """Parse GPL entry from SOFT file.
 
     Args:
         filepath (:obj:`str` or :obj:`Iterable`): Path to file with 1 GPL entry
             or list of lines representing GPL from GSE file.
         entry_name (:obj:`str`, optional): Name of the entry. By default it is
             inferred from the data.
+        partial (:obj:'iterable', optional): Accession ID of GSM that you want
```

Comment:
For the help I would go for something like that: A list of accession IDs of GSMs to be partially extracted from GPL, works only if a file/accession is a GPL?

Reply:
Yours explain of 'partial' argument is more evident than mine. I will update the explains.